api/tests/unit/import_export/test_unit_import_export_export.py

```diff
@@ -125,6 +130,50 @@ def test_export_environments(project):
     # TODO: test whether the export is importable
 
 
+def test_export_metadata(environment, organisation):
+    # Given
+    environment_type = ContentType.objects.get_for_model(environment)
+
+    metadata_field = MetadataField.objects.create(
+        name="test_field", type="int", organisation=organisation
+    )
+
+    environment_metadata_field = MetadataModelField.objects.create(
+        field=metadata_field, content_type=environment_type
+    )
+
+    metadata = Metadata.objects.create(
+        object_id=environment.id,
```

Comment:
One thing I'm keen to check here is whether this link to the environment will still work post import? Or if we could see some unexpected results here?